chapters/synchronous.tex

```diff
@@ -398,8 +398,8 @@ \section{Clock Constructors}\doublelabel{clock-constructors}
 The first input argument, \lstinline!intervalCounter!, is a clocked Component
 Expression (see \autoref{argument-restrictions-component-expression}) or a parameter expression of type
 \textbf{Integer} with min=0. The optional second argument \lstinline!resolution!
-(default=1) is a parameter expression of type Integer with min=1 and
-unit ``Hz''. If \lstinline!intervalCounter! is a parameter expression with value
+(default=1) is a parameter expression of type Integer with \lstinline!min=1! and
+\lstinline!unit! Hz. If \lstinline!intervalCounter! is a parameter expression with value
```

Comment:
I think this should be \lstinline!unit="Hz"!.

Reply:
Good idea.  Fixed.